source/dotnet/Library/AdaptiveCards.Rendering.Wpf.Xceed/XceedTimeInput.cs

```diff
@@ -32,14 +33,16 @@ public static FrameworkElement Render(AdaptiveTimeInput input, AdaptiveRenderCon
                     timePicker.Value = value;
                 TimeSpan minValue;
                 if (IsSupportedTimeFormat(input.Min) && TimeSpan.TryParse(input.Min, out minValue))
-                    timePicker.EndTime = minValue;
+                    timePicker.StartTime = minValue;
                 TimeSpan maxValue;
                 if (IsSupportedTimeFormat(input.Max) && TimeSpan.TryParse(input.Max, out maxValue))
                     timePicker.EndTime = maxValue;
                 timePicker.Watermark = input.Placeholder;
                 timePicker.Style = context.GetStyle("Adaptive.Input.Time");
```

Comment:
>"Adaptive.Input.Time") [](start = 52, length = 22)

nitnit: If you are in here and fixing it, can you add in opening and closing braces even for single line statements? Thanks!